user-guide/Reference/Soft-launch_options/Activating_SLProtocol_as_a_32_Bit_Process.md

```diff
@@ -0,0 +1,50 @@
+---
+uid: Activating_SLProtocol_as_a_32_Bit_Process
+---
+
+# Activating SLProtocol as a 32-bit process
+
+From **DataMiner 10.3.9 onwards** SLProtocol will be 64-bit by default but a fallback soft-launch has been provided.
```

Comment:
Is this the change introduced by RN 36725? If yes, could you still add this RN number in comment here? This makes it easier to track down where an RN has been added later.

In addition, we also need to make sure to always mention both the feature and the main release version. For instance, in case it's 10.4.0, this is what I'd propose:

```suggestion
From DataMiner 10.3.9/10.4.0 onwards, SLProtocol is a 64-bit process by default. However, a fallback soft-launch has been provided.<!-- RN 36725 -->
```